src/services/iiif-parser.js

```diff
@@ -440,3 +440,28 @@ export function getRenderingFiles(manifest, canvasIndex) {
   }
   return files;
 }
+
+
+/**
+ * @param {Object} manifest
+ * @return {Array} list of key value pairs for each metadata item in the manifest
+ */
+export function parseMetadata(manifest) {
+  try {
+    const metadata = parseManifest(manifest).getMetadata();
```

Comment:
Do you know if `manifesto` strips unallowed HTML tags from the metadata values?  From what I can tell it isn't.
https://iiif.io/api/presentation/3.0/#45-html-markup-in-property-values

Reply:
Yes, it seems like it doesn't strip the HTML. Maybe it's good to have this inside the parsing code. I'll check that out :+1:

Reply:
Added HTML sanitation to the metadata using `sanitize-html` JS library